src/client/components/TimeZone/TimeZone.component.js

```diff
@@ -19,15 +19,15 @@ TimeZoneDropDown.propTypes = {
   timeZones: PropTypes.arrayOf(
     shape({
       id: PropTypes.number,
-      value: PropTypes.string,
+      name: PropTypes.string,
     }),
   ),
-  timeZone: PropTypes.string,
-  setTimeZone: PropTypes.func,
+  name: PropTypes.string,
+  setName: PropTypes.func,
```

Comment:
Actually, the old name was better. Since you are picking a time zone, it is more natural to have: `<TimeZone timeZone="UTC" />` than `<TimeZone name="UTC" />`. But it is not really important.